src/modules/pages/components/ExpenditurePage/constants.ts

```diff
@@ -133,3 +137,5 @@ export const validationSchema = yup.object().shape({
     }),
   }),
 });
+
+export const EXPENDITURE_TYPE_KEY = 'expediture-type';
```

Comment:
Please, move this to the `constants` file in `src/modules/contants.ts` and name `LOCAL_STORAGE_EXPENDITURE_TYPE_KEY` - to have consistency across dapp.